apps/deploy-web/src/components/deployments/ManifestUpdate.tsx

```diff
@@ -22,21 +22,39 @@ import { deploymentData } from "@src/utils/deploymentData";
 import { getDeploymentLocalData, saveDeploymentManifest } from "@src/utils/deploymentLocalDataUtils";
 import { sendManifestToProvider } from "@src/utils/deploymentUtils";
 import { TransactionMessageData } from "@src/utils/TransactionMessageData";
+import RemoteDeployUpdate from "../remote-deploy/update/RemoteDeployUpdate";
 import { ManifestErrorSnackbar } from "../shared/ManifestErrorSnackbar";
 import { Title } from "../shared/Title";
 
 type Props = {
   deployment: DeploymentDto;
   leases: LeaseDto[];
   closeManifestEditor: () => void;
+  remoteDeploy: boolean;
+  showOutsideDeploymentMessage: boolean;
+  editedManifest: string;
+  deploymentVersion: string | null;
+  setDeploymentVersion: Dispatch<React.SetStateAction<string | null>>;
+  setEditedManifest: Dispatch<React.SetStateAction<string>>;
```

Comment:
After thinking a bit further I'm not sure it's a good idea to share state dispatchers. The outer interface on the component should basically do 2 way data binding: either accept input or callbacks for events. Otherwise it's confusing and couples components quite a bit.
Naming matters quite a bit in terms how this component would be developing in the future. I'll put an example for 1 of the props:

```suggestion
  onManifestChange: (value: string) => void;
```

so than on the client it would look like:

```tsx
<ManifestUpdate
   onManifestChange={setEditedManifest}
   {...props}
/>
```